docs/topics/mta-6-installing-web-console-on-openshift.adoc

```diff
@@ -164,3 +164,257 @@ The test results indicate that the minimum amount of memory for running {Product
 ** Installing the cluster monitoring tools requires an additional 5 GB of memory.
 ====
 
+== Red Hat Single Sign-On
+{ProductShortName} creates a basic
+https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.6[Red
+Hat Single Sign-On] instance, using the Keycloak CR of Red Hat Single
+Sign-On Operator. Keycloak is a single sign-on solution for web apps and
+RESTful web services.
+
+The {ProductShortName} Keycloak realm is fully configured by the Hub.
+
+{ProductShortName} does not enable RHSSO external access by default. However, if you
+require RHSSO, you can configure it by using the Tackle CR attribute
+`rhsso_external_access`. Otherwise, use the {ProductShortName} realm administration console to
+adjust the settings.
```

Comment:
How? What is the URL pattern for this?

Reply:
rhsso_external_access is just a boolean to enable this feature in RHSSO , the route itself (URL) will be created by RHSSO and managed by OCP,  it is dynamically created, will be different on every cluster.

Reply:
We are mixing things here. One thing is to provide a dedicated route to the RHSSO instance using that parameter in the tackle CR, and a completely different one is that even when that is not happening, the admin console for RHSSO is available at {MTA_URL}/auth.

Reply:
@rromannissen do we need to explain both of these options in this topic?